Benchmark/Benchmark/Form1.Designer.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Drawing;
+using System.Text;
+using System.Windows.Forms;
+
+namespace WindowsFormsApp2
+{
+    partial class Form1
+    {
+        /// <summary>
+        /// Required designer variable.
+        /// </summary>
+        private System.ComponentModel.IContainer components = null;
+
+        /// <summary>
+        /// Clean up any resources being used.
+        /// </summary>
+        /// <param name="disposing">true if managed resources should be disposed; otherwise, false.</param>
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && (components != null))
+            {
+                components.Dispose();
+            }
+            base.Dispose(disposing);
+        }
+
+        #region Windows Form Designer generated code
+
+        /// <summary>
+        /// Required method for Designer support - do not modify
+        /// the contents of this method with the code editor.
+        /// </summary>
+        private void InitializeComponent()
+        {
+            this.components = new System.ComponentModel.Container();
+            this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
+            this.Text = "Form1";
+        }
+        private void InitializeComponent(uint mode, uint itemsCount, uint colCount = 1, int seed = 0, int length = 8)
+        {
+            this.components = new System.ComponentModel.Container();
+            this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
+            this.Text = "Form " + itemsCount.ToString();
+            uint rowCount = itemsCount / colCount;
+            if (itemsCount % colCount != 0)
+            {
+                rowCount++;
+            }
+
+            Random r = new Random(seed);
```

Comment:
A command line option to override seed would be helpful.

Reply:
Yes, I already talked about it, it just was not so critical.

Reply:
Performance auto tests will need fixed seed for some scenarios.

Reply:
Added.